__mocks__/src/constants/index.js

```diff
@@ -0,0 +1,6 @@
+const ENV = {
+    VITE_SOLID_IDENTITY_PROVIDER: 'https://solidcommunity.net'
+};
+
+// eslint-disable-next-line

```

Comment:
Very minor nitpick (it wouldn't affect the build at all), but think we could simply add a new line for the file rather than disabling the linter with a new line?

Reply:
I don't think merges to master are the best place to review minor style issues like this. They can be picked up in the regular code review process.

That eslint-disable also isn't just disabling the end-of-file issue. It's disabling a use-default-export check because eslint would prefer this file to be `export default ENV`, however that would break the module API that's being mocked. It's odd to me that this line would disable the ending newline check as well.

I'll put up a PR to fix it though.

Reply:
Update: after testing, that line is not disabling the end of file check. eslint just isn't picking up the issue in this file at all. Not sure why that is, but it's probably an issue with the eslint config that would need some more investigation.